main/FatesInterfaceMod.F90

```diff
@@ -1932,7 +1953,15 @@ subroutine UpdateFatesRMeansTStep(sites,bc_in)
      
      type(ed_patch_type),  pointer :: cpatch
      type(ed_cohort_type), pointer :: ccohort
-     integer :: s, ifp, io_si
+     integer :: s, ifp, io_si, pft 
+     real(r8) :: new_seedling_layer_par                !seedling layer par in the current timestep
+     real(r8) :: new_seedling_layer_smp(maxpft)        !seedling layer smp in the current timestep
+     real(r8) :: new_seedling_mdd(maxpft)              !seedling layer moisture deficit days in the current timestep
+     integer  :: ilayer_seedling_root(maxpft)          !the soil layer at seedling rooting depth
```

Comment:
does this need to be a pft array?

Reply:
Yes because each pft can have a different seedling rooting depth (some seedlings develop tap roots and others don't) which makes seedling_layer_smp and seedling_mdd pft-specific.